### ISSUE TITLE
Background Services not available

### ISSUE THREAD

**Issue by JeroenTuinstra** [CastagnaIT/plugin.video.netflix#40]
*I'm submitting a ...*
  - [* ] bug report
  - [ ] feature request
  - [ ] support request

## General information
Installed the latest addon 0.14.0~beta23. Uninstalled the old one, reset the computer and installed the latest. 

### Addon version used

0.14.0~beta23

### Description

On startup it give Netflix Addon Error, background services not started. It does this with everything where a connection is needed to Netflix. In the error log, you can see that the logged-in status is none. But I don't get the chance to enter the login details anywhere, as it right away gives the Netflix Addon Error, background services not started. 

### Steps to Reproduce

1. Uninstalled old plugin.
2. Reset the computer
3. Installed the new addon. 
4. Changed in /storage/.kodi/addons/plugin.video.netflix/resources/lib/common/ipc.py line 76 from: urllib2.urlopen(url=url, data=json.dumps(data)).read(), to urllib2.urlopen(url=url, data=json.dumps(data), timeout=60).read(), as was suggested by: https://github.com/asciidisco/plugin.video.netflix/issues/658
5. Cannot login, get constant error message. 

**Expected behavior:** 
Expect a screen to appear where I can login with my details of Netflix.

**Actual behavior:**
Only gives an error message: Netflix Addon Error, background services not started

## Context (Environment)
LibreElec - latest version

### Installation

* [ *] I'm using this Netflix Repo
* [ ] I'm using other Netflix Repo (Please specify which)
* [ ] I'm using a different source (Please tell which)

### Operating System

* [ ] Linux (x86/x64)
* [ ] OSX (x86/x64)
* [ ] Windows (x86/x64)
* [* ] Linux (ARM)
* [ ] Android

#### Additional informatin on the environment
LibreELEC

## Debug log

http://ix.io/1EYP

**Comment by CastagnaIT** [comment]
Missing debug infos in log, Enable kodi debug before

**Comment by JeroenTuinstra** [comment]
I've managed to get it working somehow:
Uninstalled the plugin and installed this version: https://github.com/asciidisco/plugin.video.netflix 0.13.22
After reset it allowed me to enter a loginname and password. Netflix was working, yet very primitive. 
I then updated to this version: https://m.mediafire.com/folder/tnu4623j66c66 
After reset it asked again for my login credentials and everything seemed to work. 
This is the log file: http://ix.io/1EZ5
